CriteoPublisherSdk/Sources/Configuration/CR_DeviceInfo.m

```diff
@@ -129,12 +130,12 @@ - (NSString *)deviceId {
   return _deviceId;
 }
 
-+ (CGSize)getScreenSize {
+- (CGSize)fullScreenSize {
```

Comment:
OPT: Now this become an instance method, it could be declared as a property and used with dot notation

Reply:
Done